### ISSUE TITLE
"Cannot find browser" error when opening links

### ISSUE THREAD

**Issue by ghost** [jahirfiquitiva/Frames#199]
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests
- [X] I have verified that I am using the latest version.

### Device/App info:
 - Frames Version: `3.3.3`
 - Android version: `10 / 11`
 - Device Manufacturer: `Google`
 - Device Name: `Pixel 2 XL real/virtual`
 - Various browsers are installed, and there is a default browser

### Describe the bug
If links to TOS, privacy statement or website link in about section is clicked, a toast message appears saying "cannot find a browser" - but browsers and webview are available.

### Reproduction Steps
1. Open app and go to a section with links
2. Tap on the buttons/links
3. See message

### Expected behavior
Links open in default browser

### Screenshots
<img width="464" alt="2020-10-15 02_09_26-Frames  D__Android_Projekte_Rywall  -  _app_src_main_res_values_colors xml  ap" src="https://user-images.githubusercontent.com/21237183/96058681-30b04200-0e8c-11eb-9f34-432aa58fc383.png">

### Code and/or Logs
```   <!-- Privacy Policy link or url -->
    <string name="privacy_policy_link">"https://t-ryder.de/data-privacy/"</string>

    <!-- Terms and Conditions link or url -->
    <string name="terms_conditions_link">"https://t-ryder.de/imprint/"</string>
```

**Comment by Stuie** [comment]
I just built the app, current `master` branch (version 3.3.3), and I couldn't reproduce this. Pressing the "Website" button, for example, prompts me to choose which browser to open the link with. Testing on a Google Pixel XL running Android 10.

I inspected the code, and it seems to be doing the right thing, by checking there is a handler for the intent before firing it:

```
fun Context.openLink(url: String?) {
    val link = url ?: return
    val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(link))
    if (browserIntent.resolveActivity(packageManager) != null)
        startActivity(browserIntent)
    else toast("Cannot find a browser")
}
```

**Comment by ghost** [comment]
@Stuie just got my new device with Android 10 / oxygen os. It works here. But on the virtual taimen Wirth Android 10 and on the real taimen with latest Android 11 it did not work.

**Comment by jahirfiquitiva** [comment]
Fixed in v `3.3.5`
